sdk/servicebus/azure-mgmt-servicebus/setup.py

```diff
@@ -51,7 +51,6 @@
         'Programming Language :: Python',
```

Comment:
If you'd like to contribute to `azure-mgmt-servicebus`, please open a separate PR with all changes for that library.